src/NLog.Loki.Tests/NLog.Loki.Tests.csproj

```diff
@@ -13,8 +13,8 @@
     <PackageReference Include="Moq" Version="4.18.4" />
     <PackageReference Include="nunit" Version="3.13.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
```

Comment:
Strange with the duplicate `PackageReference`

Reply:
Thank you for pointing this out. I probably made a mistake when solving merge conflicts.